modules/knative-eventing-modifying-consumer-group-IDs-and-topic-names.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/tuning/overriding-config-eventing.adoc
+
+:_content-type: PROCEDURE
+[id="knative-eventing-modifying-consumer-group-IDs-and-topic-names_{context}"]
+= Modifying consumer group IDs and topic names
+
+You can change templates for generating consumer group IDs and topic names used by your triggers, brokers, and channels.
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {ocp-product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your {ocp-product-title} cluster.
+* You have created a project or have access to a project that has the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. To change templates for generating consumer group IDs and topic names used by your triggers, brokers, and channels, modify the `KnativeKafka` resource:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: KnativeKafka
+metadata:
+  name: <config-map-name> <1>
```

Comment:
This needs to be `name: knative-kafka` . It's not a name of some custom configmap. Should be similar to https://docs.openshift.com/serverless/1.31/install/kube-rbac-proxy-kafka.html#serverless-configuring-kube-rbac-proxy-resources-for-kafka_kube-rbac-proxy-kafka